autojustice.lua

```diff
@@ -66,7 +98,21 @@ function runScript ()
         return
     end
 
-    -- TODO: Set possible variables here
+    loadState()
+
+    argparse.processArgsGetopt(args, {
+        {'j', 'jailcitizen', hasArg=true, handler=function(arg) convictMode.citizen.jail = arg end},
```

Comment:
the `arg` passed to this function is a string that you'll have to convert to a boolean. the string "false" is truthy in Lua